.github/workflows/release-git.yml

```diff
@@ -125,6 +125,7 @@ jobs:
           artifacts-token: ${{ secrets.GITHUB_TOKEN }}
           git_artifacts_i686_workflow_run_id: ${{ env.I686_WORKFLOW_RUN_ID }}
           git_artifacts_x86_64_workflow_run_id: ${{ env.X86_64_WORKFLOW_RUN_ID }}
+          git_artifacts_aarch64_workflow_run_id: ${{ env.AARCH64_WORKFLOW_RUN_ID }}
```

Comment:
@dscho I was just going through the entire chain again, and noticed that I missed this part to pass `git_artifacts_aarch64_workflow_run_id` to the workflow.

While adding this input, I got `Invalid action input 'git_artifacts_aarch64_workflow_run_id'` in VS Code. I'm pretty sure that's because line 114 uses `git-for-windows/git-for-windows-automation/.github/actions/github-release@release` (the `release` branch), which isn't up-to-date with the latest changes from the `main` branch yet. Is that intentional?

Reply:
> While adding this input, I got `Invalid action input 'git_artifacts_aarch64_workflow_run_id'` in VS Code. I'm pretty sure that's because line 114 uses `git-for-windows/git-for-windows-automation/.github/actions/github-release@release` (the `release` branch), which isn't up-to-date with the latest changes from the `main` branch yet. Is that intentional?

Yes, this is intentional, and the `release-git` workflow tries to [always ensure that `main` and `release` start out identically](https://github.com/git-for-windows/git-for-windows-automation/blob/bea16329800cf42756827a6fab47acd73562031a/.github/workflows/release-git.yml#L37-L80).

0f5ecb3accc20fbd10e9d6c688da18ad9b0a8e3d explains the rationale a bit, and 060621ab8672ce51d78f67ed3cbc34ca45798da0 paints a fuller picture.

The tl;dr is: release automation is hard, and the idea is delusional that one could get it all right in a single YAML file that won't need to adapt to external changes, ever, or at least can be adapted correctly _before_ deploying it. Instead, the need to adapt to external changes are typically detected in the middle of a release, with half the things deployed, and the rest still needing to be deployed. Using the composite Actions from a _different_ branch allows _that_ branch to be updated and the workflow job can then be restarted _after editing just that job_.

Release automation is a lot easier in Azure DevOps' original design, without YAML. You can edit failed tasks in the web browser and then re-run from there. Highly convenient. But then there was this huge push for YAML a couple of years ago, and GitHub Actions was designed with that in mind and therefore only offers YAML _that must be part of the commit history_, that's why we're stuck with this ugly, ugly, _uggallee_ work-around. GitHub Actions simply lacks the support we need for complex deployments in the real world.